### DISCUSSION TITLE
Error: Save conflict. Another user has already modified this page

### DISCUSSION THREAD

**Discussion by HugoFMesquita** [requarks/wiki#6565] [Help / Questions]
Hey guys,

Whenever I save a published or unpublished wiki, I receive an error message followed by a conflict. If I try to close the wiki, I get another message.

"Save conflict. Another user has already modified this page."

![image](https://github.com/requarks/wiki/assets/129251407/854c3127-aec4-4d83-ad2c-3f1992fe6747)

Hosted on Heroku
Wiki.js version: 2.5.297
Linux - Alpine Linux (linux) 3.17.1 x64 
Node.js: 16.19.0
PostgreSQL: 15.3

A few people reported a similar issue (#3903, #6069, #3268) but couldn't find a solution. 

Any help is appreciated.

**Comment by NGPixel** [comment]
What's the first error message? (before the save conflict)

**Reply by HugoFMesquita** [reply]
Here is the sequence...

First message:
![image](https://github.com/requarks/wiki/assets/129251407/cb0c30b7-baf5-42dd-bc43-0d19e3a9b731)

Second message:
![image](https://github.com/requarks/wiki/assets/129251407/504affb9-aad4-47ea-b18b-042e8cf8b480)

Third message:
![image](https://github.com/requarks/wiki/assets/129251407/54ad0caf-3a13-4fd8-838d-1f25116f3373)
